app/src/main/java/com/dsu/pocapplication/news/ui/login/activity/LoginActivity.kt

```diff
@@ -0,0 +1,14 @@
+package com.dsu.pocapplication.news.ui.login.activity
```

Comment:
there is no requirement of package "activity" inside login

Reply:
done

Reply:
Login should not be a part of the news package

Reply:
This was the similar comment ive given to repository. it should be addressed here as well